app/src/main/java/ch/hikemate/app/model/route/HikeRoutesRepositoryOverpass.kt

```diff
@@ -365,9 +365,7 @@ class HikeRoutesRepositoryOverpass(private val client: OkHttpClient) : HikeRoute
           // int_name has priority over name
           "name" -> if (name == null) name = tagsReader.nextString() else tagsReader.skipValue()
           "name:en" -> nameEn = tagsReader.nextString()
-          "osmc:name",
-          "operator",
-          "symbol" ->
+          "osmc:name" ->
```

Comment:
Question: Why do you stop using `operator` or `symbol` ? Is it because they always are not interesting ?

Reply:
Exactly! I just saw that doing so leads to having multiple "Yellow diamond" (From the name of the symbol on the hike) hikes, which is (I think), worse that "Unnamed hike", especially when there is like 5 or 6 "Yellow diamond" hikes following